data_validation/combiner.py

```diff
@@ -86,18 +166,18 @@ def generate_report(
     source_pivot = _pivot_result(
         source, join_on_fields, run_metadata.validations, consts.RESULT_TYPE_SOURCE
     )
-    source_df = client.execute(source_pivot)
+    source_pivot_df = client.execute(source_pivot)
 
     target_pivot = _pivot_result(
         target, join_on_fields, run_metadata.validations, consts.RESULT_TYPE_TARGET
     )
-    target_df = client.execute(target_pivot)
+    target_pivot_df = client.execute(target_pivot)
 
     con = ibis.pandas.connect(
         {
-            consts.RESULT_TYPE_SOURCE: source_df,
+            consts.RESULT_TYPE_SOURCE: source_pivot_df,
             "differences": differences_df,
```

Comment:
At this point I think we should create a const variable for the `"differences"` string too.

Reply:
Agreed